stdlib/public/stubs/SwiftNativeNSXXXBase.mm.gyb

```diff
@@ -117,6 +133,37 @@ extern "C" int32_t swift_stdlib_compareNSStringDeterministicUnicodeCollation(
 }
 
 SWIFT_RUNTIME_STDLIB_INTERFACE
+extern "C" int32_t
+swift_stdlib_compareNSStringDeterministicUnicodeCollationStack(
+    void *baseAddressLhs, uintptr_t countAndFlagsLhs, void *ownerLhs,
+    void *baseAddressRhs, uintptr_t countAndFlagsRhs, void *ownerRhs) {
+
+  _SwiftStackAllocatedNSContiguousString stackAllocatedStringLhs = {
+      {
+          &_TMCs19_NSContiguousString, // isa.
+      },
+      baseAddressLhs,
+      countAndFlagsLhs, // core.
+      ownerLhs
+  };
+  _SwiftStackAllocatedNSContiguousString stackAllocatedStringRhs = {
+      {
+          &_TMCs19_NSContiguousString, // isa.
+      },
+      baseAddressRhs,
+      countAndFlagsRhs, // core.
+      ownerRhs
```

Comment:
IIRC indentation is still 2 spaces even in initializer lists... Could you clang-format?

Also, it seems reasonable to put `{ &_TMCs19_NSContiguousString },` on one line for compactness without hurting readability.